### PR TITLE
fix(alert): Add default action

### PR DESCRIPTION
**Pull request by luciatoma**
# Description

Added default action to `Alert` component

### CODE REVIEW

**Comment by rppld** [comment]
Imo there’s a bit too much space between the message and the action. Maybe try making it half of that?

Screenshot 2019-09-20 at 12 53 33

**Comment by rppld** [comment]
Spacing for alerts with the old `secondaryAction` is now off.

Screenshot 2019-09-20 at 12 54 52

File: src/components/Alert/index.stories.js [hunk 1]
```diff
@@ -8,6 +8,57 @@ storiesOf('Alert', module)
       <a href="https://example.com">Paypal</a> is not activated yet.
     </Alert>
   ))
+  .add('info with action', () => (
```
Comment (rppld) [inline src/components/Alert/index.stories.js]:
```suggestion
  .add('with action', () => (
```

**Review by glenngijsberts — APPROVED** [review]
🔥🔥🔥

**Review by rppld — APPROVED** [review]

**Pull request merged**